CHANGELOG.md

```diff
@@ -26,5 +30,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Fix rustdoc warnings
 - Upgrade nrf51-hal to 0.12.1
 
-[Unreleased]: https://github.com/nrf-rs/microbit/compare/v0.9.0...HEAD
+[Unreleased]: https://github.com/nrf-rs/microbit/compare/v0.10.0...HEAD
+[0.10.0]: https://github.com/nrf-rs/microbit/releases/tag/v0.10.0...v0.9.0
```

Comment:
```suggestion
[0.10.0]: https://github.com/nrf-rs/microbit/releases/tag/v0.9.0...v0.10.0
```
Close, but no cigar. 😉

Reply:
aww man! I'm doing the fixes in `xtask bump`, regenerating and then rebasing so typos are getting through. Hopefully this time. 🤞🏻